examples/blockly-rtc/src/http/index.js

```diff
@@ -23,31 +23,100 @@
  */
 
 import * as Blockly from 'blockly';
-import {getSnapshot, getEvents, writeEvents} from './workspace_client_handlers';
-import {getPositionUpdates, sendPositionUpdate} from './user_data_handlers';
+import { getSnapshot, getEvents, writeEvents } from './workspace_client_handlers';
```

Comment:
Please revert this whitespace change.

Reply:
I think this still needs to be fixed as well! Please remove the spaces inside the braces so it looks like:

```
import {getSnapshot, getEvents, writeEvents}
```